packages/@react-spectrum/color/test/ColorSlider.test.tsx

```diff
@@ -67,11 +64,11 @@ describe('ColorSlider', () => {
       expect(slider).toHaveAttribute('aria-labelledby');
       expect(slider).not.toHaveAttribute('aria-label');
 
-      let label = document.getElementById(slider.getAttribute('aria-labelledby'));
+      let label = document.getElementById(slider.getAttribute('aria-labelledby') as string);
```

Comment:
why?

Reply:
get attribute can return null, but we know it will exist, so I can just say it's a string since getElementById doesn't accept null

Reply:
maybe using the non-null assertion would be better? https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-0.html#non-null-assertion-operator

```suggestion
      let label = document.getElementById(slider.getAttribute('aria-labelledby')!);
```

Reply:
ahhhh, yeah, that seems better